package.json

```diff
@@ -55,8 +56,8 @@
     "eslint-plugin-json": "^1.2.0",
     "eslint-plugin-mocha": "^4.11.0",
     "eslint-plugin-react": "^7.1.0",
-    "istanbul": "~0.4.5",
     "mocha": "~3.2.0",
-    "npm": "~3.10.10"
+    "npm": "~3.10.10",
```

Comment:
Do we want to try to update this dep as part of this work?

Reply:
yea I can do that. its only used in tests.